webpack.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     libraryTarget: 'var'
   },
   externals: {
+    'text-encoding': 'TextEncoder',
```

Comment:
And, amusingly enough, the size of the `jose.min.js` webpack'd bundle went from 605K (with the `text-encoder` lib bundled in) to a svelte 69K. :)